examples/dreamcast/micropython/example.c

```diff
@@ -0,0 +1,79 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+#include <kos.h>
+
+#include <micropython/py/parse.h>
+#include <micropython/py/lexer.h>
+#include <micropython/py/gc.h>
+#include <micropython/py/nlr.h>
+#include <micropython/py/compile.h>
+#include <micropython/py/runtime.h>
+#include <micropython/py/stackctrl.h>
+#include <micropython/py/qstr.h>
+#include <micropython/py/obj.h>
+
+static char mp_heap[8 * 1024];
+
+static void load_module(void) {
+    nlr_buf_t nlr;
+    if (nlr_push(&nlr) == 0) {
+        mp_lexer_t *lex = mp_lexer_new_from_file(qstr_from_str("/rd/script.py"));
```

Comment:
Leaving a comment here for myself for Falco's rec to add error checking

Reply:
@gyrovorbis I remembered that this `nlr` thing in `load_module` is some setjmp black magic to throw/catch errors so the whole demo is wrapped in it now and should catch any errors thrown by micropython.